src/open_inwoner/ssd/views.py

```diff
@@ -39,15 +42,15 @@
             report_date_iso = form.data["report_date"]
             request_base_url = request.build_absolute_uri()
 
-            pdf_content = client.get_report(bsn, report_date_iso, request_base_url)
+            pdf_content = client.get_reports(bsn, report_date_iso, request_base_url)
 
             if pdf_content is None:
                 return redirect(
-                    self.request.get_full_path() + f"?report={report_date_iso}"
+                    self.request.get_full_path()
+                    + f"?report={report_date_iso}&not_found"
```

Comment:
I'm not sure if you want `request.get_full_path()` here:

    HttpRequest.get_full_path(): Returns the path, plus an appended query string, if applicable.

If you *don't* want to keep the current query string then this is the wrong method.

If you *do* want to keep the current query string then you can't just append your query with `'?'` (use `furl()` to update)

(Side note: in practice `if pdf_content is None:` is almost always better written like `if not pdf_content:` because this also catches empty strings/bytes etc).